### PR TITLE
OPS-1613 Norwegian misspellings

### PR DESCRIPTION
**Pull request by tpartyka**

### CODE REVIEW

File: languagetool-language-modules/no/src/main/java/org/languagetool/rules/no/MorfologikNorwegianSpellerRule.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+  @Override
+  protected String getIgnoreFileName() {
+    return "/no/ignore.txt";
+  }
+
+  @Override
+  public String getSpellingFileName() {
+    return "/nl/spelling.txt";
```
Comment (thduc) [inline languagetool-language-modules/no/src/main/java/org/languagetool/rules/no/MorfologikNorwegianSpellerRule.java]: nl?

File: languagetool-language-modules/no/src/main/java/org/languagetool/rules/no/MorfologikNorwegianSpellerRule.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+  @Override
+  protected String getProhibitFileName() {
+    return "/nl/prohibit.txt";
```
Comment (thduc) [inline languagetool-language-modules/no/src/main/java/org/languagetool/rules/no/MorfologikNorwegianSpellerRule.java]: also nl?

File: languagetool-language-modules/no/src/main/java/org/languagetool/rules/no/MorfologikNorwegianSpellerRule.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+  @Override
+  public String getId() {
+    return "MORFOLOGIK_RULE_NO";
+  }
+
+  @Override
+  protected String getIgnoreFileName() {
+    return "/no/ignore.txt";
```
Comment (thduc) [inline languagetool-language-modules/no/src/main/java/org/languagetool/rules/no/MorfologikNorwegianSpellerRule.java]: does the file exist?

File: languagetool-language-modules/no/src/main/java/org/languagetool/rules/no/MorfologikNorwegianSpellerRule.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+import org.languagetool.Language;
+import org.languagetool.UserConfig;
+import org.languagetool.rules.spelling.morfologik.MorfologikSpellerRule;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.ResourceBundle;
+
+public final class MorfologikNorwegianSpellerRule extends MorfologikSpellerRule {
```
Comment (thduc) [inline languagetool-language-modules/no/src/main/java/org/languagetool/rules/no/MorfologikNorwegianSpellerRule.java]: would be good to double check if Swedish (`sv`) and Danish (`da`) work (these 2 are used by our spelling checker https://github.com/Adthena/adthena-adwords/blob/dev/src/main/scala/adthena/detectors/LanguageToolDetectorService.scala) and why these 2 don't need `MorfologikSpellerRule`?

**Review by thduc — CHANGES_REQUESTED** [review]
Please check my comments.

File: languagetool-language-modules/no/src/main/java/org/languagetool/rules/no/MorfologikNorwegianSpellerRule.java (hunk 4 above)
Comment (tpartyka) [reply languagetool-language-modules/no/src/main/java/org/languagetool/rules/no/MorfologikNorwegianSpellerRule.java]: they are using hunspell rules

File: languagetool-language-modules/no/src/main/java/org/languagetool/rules/no/MorfologikNorwegianSpellerRule.java (hunk 3 above)
Comment (tpartyka) [reply languagetool-language-modules/no/src/main/java/org/languagetool/rules/no/MorfologikNorwegianSpellerRule.java]: no, its just placeholder

**Review by grishapoliakov — APPROVED** [review]

**Review by thduc — APPROVED** [review]

**Pull request merged**